omero_figure/scripts/omero/figure_scripts/Figure_To_Pdf.py

```diff
@@ -520,6 +523,24 @@ def __init__(self, pil_img, panel, crop):
         self.scale = pil_img.size[0] / crop['width']
         self.draw = ImageDraw.Draw(pil_img)
 
+        if 'border' in panel and panel['border'].get('showBorder'):
+            sw = panel['border'].get('strokeWidth')
+            shift_pos = sw / (float(panel['zoom'])/100)
```

Comment:
I'm not sure I understand the need for diving by the zoom here?
When I was testing on a big image, the zoom was very large, so the shift_pos was effectively zero and the border wasn't getting drawn around the outside of the panel. When I simply tried `shift_pos = sw` it seemed to work OK?